apps/coscrad-frontend-e2e/src/e2e/resource-index-to-detail-flows/connect-term-to-video.e2e.cy.ts

```diff
@@ -0,0 +1,130 @@
+describe(`term detail flow`, () => {
+    describe(`connect resources with note`, () => {
+        const createConnectionCommandLabel = `Create Connection with Note`;
+
+        const newNoteText = 'This is an interesting note.';
+
+        beforeEach(() => {
+            cy.visit(`Resoruces/Terms`);
+        });
+
+        describe(`when the user is logged-in as COSCRAD admin`, () => {
+            beforeEach(() => {
+                cy.login();
+
+                cy.getByDataAttribute('nav-menu-icon').click();
+
+                cy.get('[href="/Resources"] > .MuiButtonBase-root').click();
+
+                cy.getByDataAttribute('term').click();
```

Comment:
It might be nice to make this login and navigate to resource index a Cypress command.